src/com/company/FileContentsCopier.java

```diff
@@ -0,0 +1,29 @@
+package com.company;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class FileContentsCopier {
+
+    void copyContents() {
+
+        try {
+            FileReader fileReader = new FileReader("/home/webonise/IdeaProjects/Assignment-6/src/com/company/textfiles/input");
+            BufferedReader bufferedReader = new BufferedReader(fileReader);
+            FileWriter fileWriter = new FileWriter("/home/webonise/IdeaProjects/Assignment-6/src/com/company/textfiles/output", true);
+            String stringLine;
+
+            while ((stringLine = bufferedReader.readLine()) != null) {
+                fileWriter.write(stringLine + "\n");
+                fileWriter.flush();
+            }
+            bufferedReader.close();
+            fileWriter.close();

```

Comment:
also use finally block to close the resources in case if something breaks in try block and this part of code is not at all executed then the resources won't be closed properly. you can use try with resources as well here.